recipe/build.sh

```diff
@@ -11,7 +11,7 @@ then
     export LINKFLAGS="${LINKFLAGS} -stdlib=libc++ -std=c++11"
 fi
 
-./configure --prefix="${PREFIX}"
+./configure --prefix="${PREFIX}" --enable-unicode
```

Comment:
I like this idea and just wanted to check that it was working.  So, I looked in the Circle CI log and saw this.

    configure: WARNING: unrecognized options: --enable-unicode

After looking at `configure`s help, I found this option `--enable-utf`, which is what I think we need here. This blocks us from using `EBCDIC` in the future, but I don't know anybody that use that so I think we are fine.